docs/doc/30-reference/20-functions/123-file-functions/_category_.json

```diff
@@ -0,0 +1,7 @@
+{
+  "label": "File Functions",
```

Comment:
This is not a function, it's a statement.

Place this statement under https://databend.rs/doc/reference/sql/ddl

Reply:
I saw SF lists it as a part of file functions. Is it different?
https://docs.snowflake.com/en/sql-reference/functions-file.html

Reply:
Yep, we implement `presign` in a different way.